src/cmd/factory/impl/common.go

```diff
@@ -114,7 +115,10 @@ func getGCAndVerifyUser(ctx context.Context, userID string) (*connector.GraphCon
 	}
 
 	// build a graph connector
-	gc, err := connector.NewGraphConnector(ctx, graph.HTTPClient(graph.NoTimeout()), acct, connector.Users)
+	// TODO: log/print recoverable errors
+	errs := fault.New(false)
```

Comment:
in this and the other stuff in `cmd` package in this PR shouldn't we check if `errs` contains anything? We check the returned `err` from the function, but seems like we're still missing checking `errs`